shell/models/fleet.cattle.io.cluster.js

```diff
@@ -141,6 +130,31 @@ export default class FleetCluster extends SteveModel {
     };
   }
 
+  get bundleInfo() {
+    const bundlesData = {
+      ready: 0,
+      total: 0
+    };
+    const readyBundles = this.status?.display?.readyBundles;
+
+    if (readyBundles) {
+      if (readyBundles.includes('/')) {
+        const dataArr = readyBundles.split('/');
+
+        if (dataArr.length === 2 && dataArr[0].length && dataArr[1].length && parseInt(dataArr[0]) >= 0 && parseInt(dataArr[1]) >= 0) {
```

Comment:
We might be able to simplify this by eliminating the length checks for the first two indexes of `dataArr` and instead asserting that they are a number

```suggestion
        if (dataArr.length === 2 && !isNaN(dataArr[0]) && !isNaN(dataArr[1])) {
```

Reply:
Hmm.. maybe not, we're also asserting that the values are `>=0`. `parseInt` should still still evaluate to false when either index in `dataArr` is `null` or `undefined`

```suggestion
        if (dataArr.length === 2 && parseInt(dataArr[0]) >= 0 && parseInt(dataArr[1]) >= 0) {
```

Reply:
This last one I think is the correct condensed version... It passed all the unit test cases so I think were good. Thank you very much for reviewing this and for all the good tips to improve the code 🙏